core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/aggregation/AggregatorFunction.java

```diff
@@ -54,6 +55,31 @@ private static FunctionResolver avg() {
         );
     }
 
+    private static FunctionResolver count() {
+        FunctionName functionName = BuiltinFunctionName.COUNT.getName();
+        return new FunctionResolver(
+            functionName,
+            new ImmutableMap.Builder<FunctionSignature, FunctionBuilder>()
+                .put(new FunctionSignature(functionName, Collections.singletonList(ExprType.INTEGER)),
```

Comment:
Np: since the only difference is input type, this can be simplified into a for loop on input type array.

Reply:
totally agree, let's fix the bug firstly, then think how we could better define built in funciton.

Reply:
Sounds good!